adafruit_azureiot/iot_mqtt.py

```diff
@@ -326,8 +325,8 @@ def _get_device_settings(self) -> None:
     def __init__(
         self,
         callback: IoTMQTTCallback,
-        socket,
-        iface,
+        socket_pool,
```

Comment:
Change `__init__` to take in `socket_pool` (renamed) and `ssl_context`. Use it to pass to `MQTT`